src/main/java/com/adobe/commerce/cif/api/CustomerApi.java

```diff
@@ -64,7 +65,36 @@ Customer getCustomerById(
         @ApiParam(value = ACCEPT_LANGUAGE_DESC)
         @HeaderParam(ACCEPT_LANGUAGE) String acceptLanguage
     );
-    
+
+    @POST
+    @Path("/auth")
+    @ApiOperation(
+        nickname = "postAuthentication",
+        value = "Performs a authentication request to obtain a customer access token. Currently two types are " +
+            "supported: \"guest\" returns a token for a new guest (non-authenticated) customer and " +
+            "\"credentials\" authenticates a registered customer and returns a token for a successfully " +
+            "authentication otherwise results in an error"
+    )
+    @ApiResponses(value = {
+        @ApiResponse(code = HTTP_OK, message = HTTP_OK_MESSAGE, response = AuthenticationResponse.class),
+        @ApiResponse(code = HTTP_BAD_REQUEST, message = HTTP_BAD_REQUEST_MESSAGE, response = ErrorResponse.class),
+        @ApiResponse(code = HTTP_UNAUTHORIZED, message = HTTP_UNAUTHORIZED_MESSAGE, response = ErrorResponse.class)
+    })
+    @Consumes(MediaType.APPLICATION_JSON)
+    AuthenticationResponse authentication(
```

Comment:
I'd rather call the method `postAuthentication` and remove the nickname ... I don't know why we did it like that for the login.